cmd/brew-postgresql-upgrade-database.rb

```diff
@@ -1,12 +1,24 @@
-#:  * `postgresql-upgrade-database`:
+#:  * `postgresql-upgrade-database` [options]
 #:    Upgrades the database for the `postgresql` formula.
+#:
+#:    -D, --pgdata    Location of the database configuration
+#:                    files. Same as the pg_ctl -D flag.
+
+require "optparse"
+options = {}
+OptionParser.new do |opts|
+  opts.on("-D", "--pgdata DATADIR", "Location of the database configuration files.") do |data_dir|
```

Comment:
I mirrored the option names from `pg_ctl` in hopes of keeping it consistent for folks using that option. But I'm happy to use another name.